x/wasm/types/validation.go

```diff
@@ -5,8 +5,6 @@ import (
 )
 
 const (
-	MaxWasmSize = 500 * 1024
```

Comment:
Removing is dangerous.
Making this `var` could allow an app importing this to override the value at compile time, yet maintain a safety guarantee in CheckTX.

This is not the same as a module param, but better than a const.